saleor/graphql/csv/enums.py

```diff
@@ -38,6 +38,5 @@ class ProductFieldEnum(graphene.Enum):
     CHARGE_TAXES = "charge taxes"
     PRODUCT_IMAGES = "product images"
     VARIANT_SKU = "variant sku"
-    COST_PRICE = "cost price"
```

Comment:
Are we removing the ability to export cost price?

Reply:
No, we are exporting cost price from ProductVarianChannelListing, export function exporting all fields related to channel listing.

Reply:
OK, but I think the frontend now relies on this schema - in the CSV export the UI they have a checkbox "Cost price", which, when selected, sends this enum value to the export mutation. Removing it from the schema would break the export. If we want to still allow for exporting this field, I guess it should stay in the schema?